packages/carbon-web-components/src/components/data-table/table.ts

```diff
@@ -236,10 +236,17 @@ class CDSTable extends HostListenerMixin(LitElement) {
   private _handleFilterRows() {
     const unfilteredRows = [] as any;
     forEach(this._tableRows, (elem) => {
-      const rowText = elem.textContent?.trim();
-      const filtered = this.filterRows(rowText as string, this._searchValue);
+      let rowText = elem.textContent?.trim();
+      let filtered = this.filterRows(rowText as string, this._searchValue);
+      console.log('filtered', filtered);
```

Comment:
```suggestion
```

Reply:
Thanks @IgnacioBecerra.
log is removed.